src/VisualStudio/Core/Def/Implementation/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -275,5 +289,58 @@ protected static (Guid, string projectName, string? projectFlavor) GetGuidAndPro
 
             return (guid, projectName, projectFlavor);
         }
+
+        private void RemoveExistingInfoBar()
+        {
+            Contract.ThrowIfFalse(this.ThreadingContext.HasMainThread);
+
+            var infoBar = _infoBar;
+            _infoBar = null;
+            if (infoBar != null)
+            {
+                var infoBarHost = GetInfoBarHost();
+                infoBarHost?.RemoveInfoBar(infoBar);
+            }
+        }
+
+        private IVsInfoBarHost? GetInfoBarHost()
+        {
+            Contract.ThrowIfFalse(this.ThreadingContext.HasMainThread);
+
+            // Guid of the FindRefs window.  Defined here:
+            // https://devdiv.visualstudio.com/DevDiv/_git/VS?path=/src/env/ErrorList/Pkg/Guids.cs&version=GBmain&line=24
+            var guid = new Guid("a80febb4-e7e0-4147-b476-21aaf2453969");
+
+            if (_serviceProvider.GetService(typeof(SVsUIShell)) is not IVsUIShell uiShell ||
+                uiShell.FindToolWindow((uint)__VSFINDTOOLWIN.FTW_fFindFirst, ref guid, out var windowFrame) != VSConstants.S_OK ||
```

Comment:
Doesn't look like there is an `FTW_fFindLast`, but does this do the right thing when Keep Results is used?

Reply:
This is a good question.  I tried combinations of tings to see if i could break this, but it always seemed to work.  So, afaict, this is ok... but i'm not sure why TBH :)